### DISCUSSION TITLE
Youtube:)

### DISCUSSION THREAD

**Discussion by MascotWorld** [xtrime-ru/antizapret-vpn-docker#8] [Q&A] [answered]
Добавил googlevideo.com и youtube.com в include-hosts-custom и ничего не поменялось(. Есть какие-то идеи как обойти замедление? (пытался добавлять ip которые увидел через IPvFOO, уже в include-ips-custom, тоже не помогло)

**Comment by xtrime-ru** [comment]
Столкнулся с тойже проблемой пару недель назад. Почему то include-hosts-custom не срабатывает. Спрошу на форуме: https://ntc.party/c/antizapret-prostovpn-org/5

**Reply by MascotWorld** [reply]
include-hosts-custom работает, например chatgpt, который я добавил, работает. Тут наверное дело в другом, тут (https://forummikrotik.ru/viewtopic.php?t=16909) нашел инфу, что каким-то чудесным образом запрос к ютубу обратно подает в россию (если правильно понял). Через tracert посмотрел айпи, добавил его в include-ips-custom
![image](https://github.com/user-attachments/assets/7712839e-9180-41fb-b893-d0136f1ce6f0) стало вот так 
![image](https://github.com/user-attachments/assets/b7617c78-7188-4277-91c7-ad440ab584d0)

**Reply by xtrime-ru** [reply]
Нашел в чем причина. Внутри контейнера в файле /root/antizapret/config/exclude-hosts-dist.txt лежит список доменов, которые не надо гнать через антизапрет. 

Сейчас почищу его и будет работать!

**Comment by xtrime-ru** [comment, marked as answer]
Исправил в коммите https://github.com/xtrime-ru/antizapret-vpn-docker/commit/2223ab46ea7f51f0cf58e6531a8c7dbd9f0efe32: 

Заменил файл /root/antizapret/config/exclude-hosts-dist.txt на ссылку на пустой файл. В нем исключался youtube.com и некоторые другие домены.

**Reply by MascotWorld** [reply]
Подскажи как склонировать, чтоб конфиги не менять?

**Reply by xtrime-ru** [reply]
Да, я перенес txt файлы в папку config, так как их уже оч много. Надо скопировать свои изменения, потом сделать git fetch && git reset --hard origin/master, потом вернуть изменения в папку config.  chatgpt, я кстати, добавил.

**Reply by MascotWorld** [reply]
Благодарю <3
